test/active_record_doctor/detectors/missing_foreign_keys_test.rb

```diff
@@ -21,6 +21,26 @@ def test_present_foreign_key_is_not_reported
     refute_problems
   end
 
+  def test_non_integer_missing_foreign_key_is_reported
+    Context.create_table(:users) do |t|
+      t.string :external_id
+    end
+
+    refute_problems
+  end
+
+  def test_uuid_missing_foreign_key_is_reported
+    skip("#{current_adapter} doesn't support materialized views") if !postgresql?
```

Comment:
It seems this error message came from another test case. Let's use a `require_*` function from `setup.rb` here (or define one if none of the existing ones make sense).

Reply:
Oops! Changed.